napari_skimage_regionprops/_table.py

```diff
@@ -100,6 +105,36 @@ def update_content(self):
         """
         self.set_content(self._layer.properties)
 
+    def append_content(self, table: Union[dict, DataFrame], how: str = 'outer'):
+        """
+        Append data to table.
+        The input table can be a pandas dataframe or a dictionary.
+        Parameters
+        ----------
+        table : Union[dict, DataFrame]
+            New data to be appended.
+        how : str, OPTIONAL
+            Method how to join the data. See also https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.merge.html
+        Returns
+        -------
+        None.
+        """
+        # Check input type
+        if type(table) == dict:
```

Comment:
```suggestion
        if not isinstance(table, DataFrame):
```
How about this? It would be more flexible and also allows input types that are derived from `dict`, not just dicts.